src/Android/Xamarin.Android/Samples/Network Analysis/ClosestFacility/ClosestFacility.cs

```diff
@@ -0,0 +1,207 @@
+                new Facility(new MapPoint(-1.3049023883956768E7, 3861993.789732541, _spatialReference))
+                };
+
+            // Center the map on the San Diego facilities.
+            Envelope fullExtent = GeometryEngine.CombineExtents(_facilities.Select(facility => facility.Geometry));
+            _myMapView.SetViewpointGeometryAsync(fullExtent, 50);
+
+            // Create a symbol for displaying facilities.
+            _facilitySymbol = new PictureMarkerSymbol(new Uri("http://static.arcgis.com/images/Symbols/SafetyHealth/Hospital.png"));
+            _facilitySymbol.Height = 30;
+            _facilitySymbol.Width = 30;
+
+            // Incident symbol
+            _incidentSymbol = new SimpleMarkerSymbol(SimpleMarkerSymbolStyle.Cross, System.Drawing.Color.FromArgb(255, 0, 0, 0), 30);
+
+            // Route to hospital symbol
+            _routeSymbol = new SimpleLineSymbol(SimpleLineSymbolStyle.Solid, System.Drawing.Color.FromArgb(255, 0, 0, 255), 5.0f);
+
+            // Create Graphics Overlays for incidents and facilities.
+            _incidentGraphicsOverlay = new GraphicsOverlay();
+            _facilityGraphicsOverlay = new GraphicsOverlay();
+
+            // Create a graphic and add to graphics overlay for each facility.
+            foreach (Facility facility in _facilities)
+            {
+                _facilityGraphicsOverlay.Graphics.Add(new Graphic(facility.Geometry, _facilitySymbol));
+            }
+
+            // Add each graphics overlay to MyMapView.
+            _myMapView.GraphicsOverlays.Add(_incidentGraphicsOverlay);
+            _myMapView.GraphicsOverlays.Add(_facilityGraphicsOverlay);
+        }
+
+        private void OnDrawStatusChanged(object sender, DrawStatusChangedEventArgs e)
+        {
+            if (e.Status == DrawStatus.Completed)
+            {
+                // Link the action of tapping on the map with the MyMapView_GeoViewTapped method.
+                _myMapView.GeoViewTapped += MyMapView_GeoViewTapped;
+
+                // Remove this method from DrawStatusChanged events.
+                _myMapView.DrawStatusChanged -= OnDrawStatusChanged;
+            }
+        }
+
+        private void MyMapView_GeoViewTapped(object sender, Esri.ArcGISRuntime.UI.Controls.GeoViewInputEventArgs e)
+        {
+            // Clear any prior incident and routes from the graphics.
+            _incidentGraphicsOverlay.Graphics.Clear();
+
+            // Create a MapPoint where the user clicked.
+            _incidentPoint = _myMapView.ScreenToLocation(e.Position);
+
+            // Populate the facility parameters than solve using the task.
+            PopulateParametersAndSolveRouteAsync();
+        }
+
+        private async void PopulateParametersAndSolveRouteAsync()
+        {
+            // Set facilities and incident in parameters.
+            ClosestFacilityParameters _closestFacilityParameters = await _task.CreateDefaultParametersAsync();
```

Comment:
I recommend removing the underscore prefix for consistency here. We usually only use the underscore for fields and not local variables.